clang/src/fileio.s

```diff
@@ -6,20 +6,41 @@ ptr1:
         .ds.b 1 ;; 1 byte
 ptr2:
         .ds.b 1
+tmp1:
+        .ds.b 1
 tmp2:
         .ds.b 1
 tmp3:
         .ds.b 1
+sp:
+        .ds.b 1
 
-.section code
+    .section code,"a"
 
 mega65_io_enable:
 	lda #$47
 	sta $d02f
 	lda #$53
 	sta $d02f
 	rts
-
+cc65_args_read_ptr1_16:	
```

Comment:
Just curious who calls cc65_args_read_ptr1_16 and cc65_args_read_tmp1_8?
(never dug too deep in this area before :))

Reply:
I noted that they were in the original `fileio.s`, but for unknown reasons or I left it out when making the clang version some time ago. I do not know how they are used, but I will check a bit more.